modules/zed/hm.nix

```
@@ -0,0 +1,31 @@
+{
+  inputs,
+  config,
+  lib,
+  ...
+}: let 
+  theme = config.lib.stylix.colors {
+    templateRepo = config.lib.stylix.templates.tinted-zed;
+  };
+in {
+  options.stylix.targets.zed = {
+    enable = config.lib.stylix.mkEnableTarget "zed" true;
+  };
+
+  config = lib.mkIf (config.stylix.enable && config.stylix.targets.zed.enable && config.programs.zed-editor.enable) {
+    programs.zed-editor = {
+      userSettings = {
```

Consider simplifying the attribute set declaration:

```suggestion
    programs.zed-editor.userSettings = {
```